src/components/ProfileView.tsx

```
@@ -88,469 +58,252 @@ const ProfileView: React.FC<ProfileViewProps> = ({
+            {/* Settings — только для своего профиля */}
+            {isOwnProfile && (
+              <button
+                onClick={() => setShowSettings(true)}
+                className="absolute top-20 right-4 md:top-24 md:right-6 z-20 p-2 rounded-xl transition-all duration-200 hover:scale-110 active:scale-95"
+                style={{
+                  background: 'rgba(255, 255, 255, 0.08)',
+                  border: '1px solid rgba(255, 255, 255, 0.12)',
+                  color: 'rgba(220, 235, 245, 0.9)',
+                }}
+                title={t('profile.settings.title')}
+              >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)